spec/controllers/locales_controller_spec.rb

```diff
@@ -50,7 +50,8 @@
 
       it "redirects to the user page" do
         put :update
-        it_redirects_to user_path(user)
+        it_redirects_to_with_error user_path(user),
```

Comment:
Could these be rewritten with parentheses to be consistent with the other examples? Also, the descriptions should mention the flash errors.